Sources/Private/Utility/Primitives/BezierPath.swift

```diff
@@ -10,7 +10,7 @@ import CoreGraphics
 // MARK: - BezierPath
 
 /// A container that holds instructions for creating a single, unbroken Bezier Path.
-struct BezierPath {
+public struct BezierPath {
```

Comment:
Instead of making `BezierPath` public, can we add an `Interpolatable` conformance to `CGPath`?

Reply:
From what I gathered, I need the `valueType` of the `ValueProvider` to return `BezierPath`, because that's the underlying type of the property to which it compares itself.

Similarly, I see that `LottieVector3D` from `PointValueProvider` is public.